RuntimeException: org.lwjgl.LWJGLException: Could not locate OpenAL library.

**Describe the bug**
A clear and concise description of what the bug is.

***Error in application***
```
Uncaught exception thrown in Thread[jME3 Main,5,main]
RuntimeException: org.lwjgl.LWJGLException: Could not locate OpenAL library.
```

***Last `error.log` stack trace***
```
Mar 17, 2022 3:56:26 PM org.wysko.midis2jam2.gui.UpdateChecker checkForUpdates$lambda-2
INFO: Checking for updates.
Mar 17, 2022 3:56:31 PM org.wysko.midis2jam2.gui.UpdateChecker checkForUpdates$lambda-2
INFO: Up to date.
Mar 17, 2022 3:56:46 PM com.jme3.system.JmeDesktopSystem initialize
INFO: Running on jMonkeyEngine 3.4.1-stable
 * Branch: HEAD
 * Git Hash: 662c8fe
 * Build Date: 2021-12-04
Mar 17, 2022 3:56:47 PM com.jme3.renderer.opengl.GLRenderer loadCapabilitiesCommon
INFO: OpenGL Renderer Information
 * Vendor: Intel
 * Renderer: Intel(R) UHD Graphics 630
 * OpenGL Version: 4.6.0 - Build 30.0.101.1219
 * GLSL Version: 4.60 - Build 30.0.101.1219
 * Profile: Compatibility
Mar 17, 2022 3:56:47 PM com.jme3.system.lwjgl.LwjglContext printContextInitInfo
INFO: LWJGL 2.9.3 context running on thread jME3 Main
 * Graphics Adapter: igdumdim64
 * Driver Version: null
 * Scaling Factor: 1
Mar 17, 2022 3:56:47 PM com.jme3.asset.AssetConfig loadText
WARNING: Cannot find loader com.jme3.audio.plugins.OGGLoader
Mar 17, 2022 3:56:47 PM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 3 more retries remaining.

Mar 17, 2022 3:56:47 PM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 2 more retries remaining.

Mar 17, 2022 3:56:47 PM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 1 more retry remaining.

Mar 17, 2022 3:56:48 PM com.jme3.audio.lwjgl.LwjglALC createALC
WARNING: Caught an LWJGLException from AL.create(). Will retry after 100 msec, with 0 more retries remaining.

Mar 17, 2022 3:56:48 PM com.jme3.app.LegacyApplication handleError
SEVERE: Uncaught exception thrown in Thread[jME3 Main,5,main]
java.lang.RuntimeException: org.lwjgl.LWJGLException: Could not locate OpenAL library.
	at com.jme3.audio.lwjgl.LwjglALC.createALC(LwjglALC.java:32)
	at com.jme3.audio.openal.ALAudioRenderer.initOpenAL(ALAudioRenderer.java:94)
	at com.jme3.audio.openal.ALAudioRenderer.initialize(ALAudioRenderer.java:229)
	at com.jme3.app.LegacyApplication.initAudio(LegacyApplication.java:292)
	at com.jme3.app.LegacyApplication.initialize(LegacyApplication.java:642)
	at com.jme3.app.SimpleApplication.initialize(SimpleApplication.java:198)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.initInThread(LwjglAbstractDisplay.java:139)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:221)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.lwjgl.LWJGLException: Could not locate OpenAL library.
	at org.lwjgl.openal.AL.create(AL.java:156)
	at org.lwjgl.openal.AL.create(AL.java:102)
	at org.lwjgl.openal.AL.create(AL.java:206)
	at com.jme3.audio.lwjgl.LwjglALC.createALC(LwjglALC.java:27)
	... 8 more


```

probably race condition?

**Offending MIDI file**
<!--Upload the MIDI file that causes the bug here.-->

none

**Expected behavior**
<!--A clear and concise description of what you expected to happen.-->

MIDI visualization began normally

**Screenshots**
<!--If applicable, add screenshots to help explain your problem.-->

![werror in app midis2jam2 openal moveaway](https://user-images.githubusercontent.com/7281060/158972322-40027732-6041-4881-83c2-9ae9a37d9cbe.png)

![wtf midis2jam2 openal is already extracted](https://user-images.githubusercontent.com/7281060/158972743-afad0d8e-238f-4ef6-94fe-a856dc3ef40f.png)
the `lwjgl.dll` & `OpenAL64.dll` are extracted by this software automatically everytime press `Start` and placed next to the JAR file, yet still cannot find it. weird.

**System**
Provide your OS, and computer specs if applicable.

ASUS ROG GL503GE Laptop:
OS = Windows 11 Pro 22H2 Build 22000.556
CPU = Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz   2.21 GHz
GPU = Intel UHD 630 + Nvidia GeForce GTX 1050 Ti
RAM = 8,00 GB (1 stick SODIMM 2667 MHz)

**Additional context**
<!--Add any other context about the problem here.-->

***Viewed article (not working, way too off relevancy, not leading to solution, etc):***
- https://stackoverflow.com/questions/29179625/could-not-locate-openal-library
- https://github.com/jMonkeyEngine/jmonkeyengine/issues/1383
- https://hub.jmonkeyengine.org/t/solved-exception-could-not-locate-openal-library/43423
- http://forum.lwjgl.org/index.php?topic=3253.0
- https://jvm-gaming.org/t/libgdx-lwjgl-could-not-locate-openal-library/42335
- https://bz.apache.org/netbeans/show_bug.cgi?id=233342
- https://community.gitpod.io/t/lwjgl-openal/1105

***Tried other methods not working:***
- Portable JAR file with Adoptium OpenJDK 11, same error
- Administrator Run the EXE file, same error
- Portable JAR file with Oracle JRE 8, incompatible! could not find main class
- Portable JAR file using latest [Manjaro XFCE edtion](https://manjaro.org) Live CD as of 17 March 2022 (version 21.2.4) and straight install OpenJDK 11 without `pacman -Syu`, assertion error, forgot assertion

***Question:***
- does Gervill uses LWJGL's audio? I believe it's not. same with MIDI output devices. perhaps if we just disable LWJGL audio renderer for the 3D part might work?

--- a/src/main/kotlin/org/wysko/midis2jam2/starter/Execution.kt
+++ b/src/main/kotlin/org/wysko/midis2jam2/starter/Execution.kt
@@ -274,6 +274,7 @@ private val defaultSettings = AppSettings(true).apply {
         ImageIO.read(this::class.java.getResource(it))
     }.toTypedArray()
     title = "midis2jam2"
+    audioRenderer = null
 }
 
 private open class StandardExecution(

